### ISSUE TITLE
A little bug when checking for new music

### ISSUE THREAD

**Issue by ephemlw** [Monika-After-Story/MonikaModDev#9299]
If you're adding a new song and asking Monika to check the music folder, she'll act as if she didn't find anything. However, that new song will actually appear in the Music section.

**Comment by Ant9cp** [comment]
If the game is closed and you add a new song in the custom_bgm folder, then you do not need to ask her to check for new music, unless if your game is still on and you add a new song in the folder, then you do need to ask her to check for new music, 


(So it's not a bug)

**Comment by ephemlw** [comment]
Yeah, I know, but it's weird that she says she can't find anything and after that adds a new song.
